src/main/resources/i18n/outdoor/common.properties

```diff
@@ -238,6 +238,11 @@ GETSTOCKBYITEMID_DESCRIPTION=Unary call which sends item id and receives stock q
 GETITEMSINSTOCK_DESCRIPTION=Server streaming method which gets information of all items in stock in stream
 UPDATEITEMSINSTOCK_DESCRIPTION=Bidirectional streaming method which takes information to update item stock and responds with updated item information
 UI_BACKEND_COMMUNICATION=UI/backend communication:
+KAFKA_ADDITIONAL_INFO_FOR_PARTITION=All messages from this application are sent to the partition 0 in the topic, it means the consumer should listen on the partition 0 of the topic.
```

Comment:
I only add the messages fo outdoor. I will localize these messages and apply them for other industries after the pull request is approved.

Reply:
These messages need to be updated after the other comments are fixed.

Reply:
I only updated the readme file, after all changes in readme file are approved, I will change these messages.